web-ui/src/main/resources/catalog/components/common/map/mapService.js

```diff
@@ -1980,8 +1974,7 @@
          * appear in the layer manager
          */
         selected: function(layer) {
-          return layer.displayInLayerManager && !layer.get('fromWps') &&
-              (!layer.get('errors') || !layer.get('errors').length);
+          return layer.displayInLayerManager && !layer.get('fromWps');
```

Comment:
Layers are shown anyway, but as they don't appear on LayerManager there is no way to handle them (or remove them). Better to show them with errors, as with this change, so the user can: 
1.- Check the list of errors associated to it
2.- If there is something added to the map and shown, even if it is ugly and weird, have a button to remove it.